docker-compose.yml

```diff
@@ -1,18 +1,29 @@
-version: "3"
 services:
   full_text_searcher:
-    container_name: full_text_searcher
-    image: full-text-searcher
+    build:
+      context: .
+      target: runtime
+      dockerfile: Dockerfile
+      args:
+        UID: ${UID:-1000}
+        GID: ${GID:-1000}
+        DEV: ${DEV:-false}
+        POETRY_VERSION: ${POETRY_VERSION:-1.5.1}
+    env_file:
+      - .env
+    volumes:
+      - .:/app
+    stdin_open: true
     depends_on:
       solr-lss-dev:
         condition: service_healthy
     tty: true
+    container_name: full_text_searcher
```

Comment:
I think if we add `network_mode: host` right under here then we should be able to connect to production solr via `http://localhost:8081` (using ssh tunneling)

Reply:
That said we can also run it outside docker so long as we have a python runtime -- the whole thing is simple enough -- and if/when HTRC uses it, they would be connecting via the actual hostname rather than using ssh tunneling and localhost

Reply:
Maybe it's also unclear here that we need to actually set up an ssh tunnel as well, e.g. `ssh -L8081:localhost:8081 squishee-1.umdl.umich.edu` (while on the library VPN)

Reply:
I will try it. I will also add to the Readme file the need to set up the ssh tunnel as well

Reply:
I have tried the suggestions but I can not access to prod inside the docker

This is the command I am using 

`docker compose exec full_text_searcher python ht_full_text_search/export_all_results.py '"justice blame"'`

I am in the VPN and also set up the ssh tunnel

See the error
![image](https://github.com/user-attachments/assets/899afdb7-10f1-4ac0-bc7a-765887555a2e)

Reply:
Maybe @moseshll or I can give it a try on Mac and see if we can get it to work and if so what needs to be done. It could also potentially be some kind of firewall issue - I've run into that on Linux before.